src/cargo/sources/git/utils.rs

```diff
@@ -296,52 +294,61 @@ impl<'a> GitCheckout<'a> {
             info!("update submodules for: {:?}", repo.workdir().unwrap());
 
             for mut child in repo.submodules()?.into_iter() {
-                child.init(false)?;
-                let url = child.url().chain_error(|| {
-                    internal("non-utf8 url for submodule")
+                update_submodule(repo, &mut child, cargo_config).chain_error(|| {
+                    human(format!("Failed to update submodule `{}`",
```

Comment:
Idiomatically cargo errors start with a lowercase letters, could that be updated here?